src/components/form-elements/_form-elements.scss

```diff
@@ -123,6 +128,7 @@ $include-html: false !default;
     height: $inputHeight;
     line-height: $inputHeight;
     appearance: none;
+    transition: border 300ms;
```

Comment:
unrelated change, border color smoothly transitions between colors now - it's subtle but looks much nicer than a sudden color change